Libraries/PushNotificationIOS/RCTPushNotificationManager.m

```diff
@@ -57,6 +57,28 @@ + (UILocalNotification *)UILocalNotification:(id)json
 
 @implementation RCTPushNotificationManager
 
++ (NSMutableDictionary *)formatLocalNotification:(UILocalNotification *)notification
```

Comment:
This doesn't use self, so you can just make it a static method. It's also better to prefer immutable objects, so return `NSDictionary` here instead of `NSMutableDictionary` as type.

```
static NSDictionary *formatLocalNotification(UILocalNotification *notification)
```